builtins/amp-img.md

```diff
@@ -77,16 +77,20 @@ If the resource requested by the `amp-img` component fails to load, the space wi
 #### Example: Specifying a fallback image
 In the following example, if the browser doesn't support WebP, the fallback JPG image displays:
 
-<div>
-<amp-iframe height="271"
-            layout="fixed-height"
-            sandbox="allow-scripts allow-forms allow-same-origin"
-            resizable
-            src="https://ampproject-b5f4c.firebaseapp.com/examples/ampimg.fallback.embed.html">
-  <div overflow tabindex="0" role="button" aria-label="Show more">Show full code</div>
-  <div placeholder></div>
-</amp-iframe>
-</div>
+[example preview="inline" playground="true"]
+```html
+<amp-img alt="Mountains"
+  width="550"
+  height="368"
+  src="{{server_for_email}}/static/inline-examples/images/mountains.webp">
```

Comment:
Where does the `server_for_email` variable come from?

Reply:
It's retrieved from the Podspec when the example is being generated and is using the preview host: `doc.pod.get_podspec().yaml.get('base_urls').get('preview')`.